scoring_harness/challenge.py

```diff
@@ -100,6 +104,21 @@ def validate_single_submission(syn, submission, status,
 
     status.status = "VALIDATED" if is_valid else "INVALID"
 
+    # Remove submission if invalid
+    if remove_cache and status.status == "INVALID":
```

Comment:
Even though this makes sense for us, if someone is only going to validate a file instead of score, the removal of cached results should still happen regardless of the status.  The issue with this is then the file will be downloaded again in the scoring step....

Reply:
Oh you're right -- I totally missed the comment about the submission getting fetched again in the scoring function.

Okay, will remove submissions regardless of status.